studybuddies/src/app/tutor/class-list-tutor/class-list-tutor.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ClassListTutorComponent } from './class-list-tutor.component';
+
+describe('ClassListTutorComponent', () => {
+  let component: ClassListTutorComponent;
+  let fixture: ComponentFixture<ClassListTutorComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ ClassListTutorComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ClassListTutorComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7155411)
```suggestion
  it("should create", () => {
```